packages/tre/src/plugins/core/index.ts

```diff
@@ -89,12 +93,7 @@ export const SCRIPT_CUSTOM_SERVICE = `addEventListener("fetch", (event) => {
   event.respondWith(${BINDING_SERVICE_LOOPBACK}.fetch(request));
 })`;
 
-const now = new Date();
-const fallbackCompatibilityDate = [
-  now.getFullYear(),
-  (now.getMonth() + 1).toString().padStart(2, "0"),
-  now.getDate().toString().padStart(2, "0"),
-].join("-");
+const FALLBACK_COMPATIBILITY_DATE = "2000-01-01";
```

Comment:
Why are we shifting from today to a compat date of 2000? Surely keeping it up to date would be better?

Reply:
Sure, that makes sense. Do we not want something slightly more recent than 2000 though?

Reply:
At least after the last behaviour change, which looks like it was `2022-08-04` https://developers.cloudflare.com/workers/platform/compatibility-dates/